python/tvm/relay/transform/transform.py

```diff
@@ -219,6 +219,18 @@ def DeadCodeElimination(inline_once=False):
     """
     return _ffi_api.DeadCodeElimination(inline_once)
 
+def GradientCell():
```

Comment:
Is gradient cell really the right name for this given it is just making 1/0s lazy.

Reply:
renamed to `LazyGradientInit`, highlight that this pass should be used after reverse mode ad to lazily initiate tensor gradients